skyportal/handlers/api/annotation.py

```diff
@@ -30,18 +49,64 @@ def get(self, annotation_id):
               application/json:
                 schema: Error
         """
-        annotation = Annotation.get_if_accessible_by(
-            annotation_id, self.current_user, raise_if_none=True
-        )
+
+        try:
+            annotation_id = int(annotation_id)
+        except (TypeError, ValueError):
+            return self.error("Must provide a valid annotation ID. ")
+        # the default is to annotate an object
+        if associated_resource_type.lower() in (
```

Comment:
Why four options here?  And when is this value `object` or `objects`?

Reply:
In principle you could comment on an object that was not saved as a source... we don't have an API for that yet but I can imagine needing it in the future.

Reply:
It's good to think ahead!  In general, we try to have a design that can be expanded in the future, but to implement only the minimal changes necessary.  Can we take out anything that is not used right now?

Reply:
OK